salt/modules/win_lgpo.py

```diff
@@ -336,7 +336,7 @@ def __init__(self):
             0: "No auditing",
             1: "Success",
             2: "Failure",
-            3: "Success, Failure",
+            3: "Success and Failure",
```

Comment:
Seems this would require users to update their states. Can we prevent that from occurring?